packages/swr/src/index.ts

```diff
@@ -235,6 +236,12 @@ const generateSwrImplementation = ({
   doc?: string;
 }) => {
   const swrProps = toObjectString(props, 'implementation');
+
+  const hasParamNameQuery = props.some(
```

Comment:
I wonder if we should have a CONST array of reservedWords and right now it would only be `['query']` that you could check but that way it would make this more flexible in the future if we found another reserved word we want to underscore?

Reply:
indeed. I'm not sure if the list will increase in the future, but I think it's a good idea in terms of emphasizing it. I'll deal with it.

Reply:
@melloware 
thank you for review. I thought about the comment for a while.

If new reservations are added in the future, you will need to validate `prop` for each variable name and give `_` to that variable. In the end, it is necessary to process in units of reserved words, and it may not be possible to be flexible. I think it's great to emphasize what happens after a reservation is made, but it actually increases the complexity at this point, so I thought it would be better to leave it as is for now.

What do you think after hearing my opinion?